### ISSUE TITLE
use strict directive requires lexically unbroken string

### ISSUE THREAD

**Issue by IgnoredAmbience** [resource-reasoning/JS_Parser#5]
tests/test262/ch14/14.1/14.1-4-s.js tests for

``` javascript
'use str\
ict';
```

Esprima outputs:

``` javascript
                                {
                                    "type": "ExpressionStatement",
                                    "expression": {
                                        "type": "Literal",
                                        "value": "use strict",
                                        "raw": "'use str\\\r\nict'",
                                        "range": [
                                            579,
                                            594
                                        ]
                                    }
```

We're using the value instead of raw_value?

Need to check if the same applies to the closure variant.

**Comment by edgemaster** [comment]
Blocking resource-reasoning/jscert_dev#35

**Comment by giltho** [comment]
This was fixed by #10
